src/components/tab-header.tsx

```diff
@@ -1,23 +1,11 @@
 import React from 'react';
-import {StyleSheet} from 'react-native';
 import {Text} from './ui';
 import {HeaderTitleProps} from '@react-navigation/elements';
-import {TEXT_BASE_1} from '../variables';
 
-// @ts-ignore
-export const TabHeader = ({headerTitle}: HeaderTitleProps) => {
-  return (
-    <Text t8 style={page.t8}>
-      {headerTitle}
-    </Text>
-  );
-};
+interface HeaderTitleT extends HeaderTitleProps {
+  headerTitle?: string;
+}
 
-const page = StyleSheet.create({
-  t8: {
-    fontStyle: 'normal',
-    fontWeight: '600',
-    textAlign: 'center',
-    color: TEXT_BASE_1,
-  },
-});
+export const TabHeader = ({headerTitle}: HeaderTitleT) => {
```

Comment:
IMHO these component useless and we can use `<Text t8>{headerTitle}</Text>` in place